index.js

```diff
@@ -0,0 +1,65 @@
+const tilesContainer = document.querySelector(".tiles");
+
+//memotest images
+
+const feelingsImgLink = {
+    happy: "images/feelings/happy.png",
+    sad: "images/feelings/sad.png", 
+    angry: "images/feelings/angry.png",
+    hungry: "images/feelings/hungry.png",
+    thirsty: "images/feelings/thirsty.png",
+    scared: "images/feelings/scared.png",
+}
+
+const feelingsImages = {}; //object that contains images to append
```

Comment:
You're absolutely right, Nico. I created that object first to use it later and never did because I created that card img element whose src dynamically changes depending on the feeling. Thank you so much for reviewing my code with such a sharp eye, I truly appreciate it <3